clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-15-vexxhost-hybrid_clusterpool.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    version: "4.15"
+    version_lower: 4.15.0-0
+    version_upper: 4.16.0-0
+  name: oko-op-06
+  namespace: openstack-k8s-operators-cluster-pool
+spec:
+  baseDomain: oooci.ccitredhat.com
+  hibernationConfig: null
+  imageSetRef:
+    name: ocp-release-4.15.14-x86-64-for-4.15.0-0-to-4.16.0-0
```

Comment:
correct: https://github.com/openshift/release/blob/fe75ae3f21467629e6846e6cb4243ca157ab10dc/clusters/hosted-mgmt/hive/pools/ocp-release-4.15.14-x86-64-for-4.15.0-0-to-4.16.0-0_clusterimageset.yaml#L8